app/questionnaire/placeholder_transforms.py

```diff
@@ -315,3 +315,9 @@ def option_label_from_value(self, value: str, answer_id: str) -> str:
         label may be simple string or a resolved placeholder string
         """
         return self.ops_helper.get_option_label_from_value(value, answer_id)
+
+    @staticmethod
+    def conditional_trad_as(trad_as: str) -> str:
```

Comment:
`trad_as `can be optional
```suggestion
    def conditional_trad_as(trad_as: Optional[str]) -> str:
```